src/common/encryption/Base64.cpp

```diff
@@ -0,0 +1,118 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "common/encryption/Base64.h"
+#include <cctype>
+
+namespace nebula {
+namespace encryption {
+
+const char Base64::encodeTab_[] =
+    "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+
+const int Base64::decodeTab_[] = {
+    -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
+    -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
+    -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
+    -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
+    -1, -1, -1, 62,    // '+'
+    -1, -1, -1, 63,    // '/'
```

Comment:
Why not use `proxygen::base64Encode() ` ？

Reply:
You reminded me yesterday.
I went to proxygen and took a look. I felt that proxygen was available, but the best one felt like using it.
Reference proxygen notes are as follows:
```
// This is a wrapper around openssl base64 encoding.  It is a temporary wrapper
// around openssl, until a more optimized version lands in folly

class Base64 {
  public:
    static std::string decode(const std::string& b64message, int padding);
    static std::string urlDecode(const std::string& b64message);
    static std::string encode(folly::ByteRange buffer);
    static std::string urlEncode(folly::ByteRange buffer);
};
```

Reply:
Can follow the progress of folly